docs/devel/architecture.md

```diff
@@ -0,0 +1,44 @@
+# karydia Concepts
+## Secure by Default
+The mission of karydia is to configure kubernetes in a more secure way. Many kubernetes default settings are unsecure. When karydia is installed these defaults are revised and supervised. Most karydia features can be switched on centrally and configured on namespace or pod level.  
+
+## Configuration
+karydia provides the possibility to select and configure the security features which should be used and supervised in a kubernetes cluster. If these settings shall be adapted for a selected namespace or pod it is possible to overwrite it with an annotation.
+
+![Configuration of karydia](https://github.com/karydia/karydia/blob/master/install/charts/values.yaml)
```

Comment:
according to [MARKDOWN SYNTAX](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf) this format with the `! exclamation mark` is only for images and you're referring a YAML file

Reply:
Would it be possible to use a relative path instead of an absolute one?